storage/database/how-to/connect-database-private-network.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: How to connect a Database Instance to a Private Network
+description: This page explains how to connect a Database Instance to a Private Network
+dates:
+  validation: 2021-11-15
+  posted: 2019-11-15
+---
+
+[Private Networks](/network/vpc/quickstart/) allows you to configure your own Layer-2 network with IPv4 adresses. Thanks to this feature, you enhance the security of your systems architecture by isolating it from the Internet. 
+
+This feature comes with a significant change in the architecture of Scaleway's managed Databases: when using Private Network, your application connects directly to your Database nodes, without going through a Load Balancer. 
+This unleashes the DB performances by reducing the latency between your nodes and increases the security of your databases, as Instances in your Private Network can directly to your Database Instance, without passing over the public Internet.
+
+You can create a new Database Instance on a Private Network or connect an existing Database Instance by adding a new Private Network Endpoint on it. 
```

Comment:
```suggestion
You can create new Database Instances to attach to your Private Network, or attach existing Database Instances by adding Private Networks endpoints to them. 
```

Reply:
Maybe it would also be good to point out here that (if I've understood correctly) the Private Network can have Instances as well as Databsaes attached to it?